lighthouse-core/computed/metrics/responsiveness.js

```diff
@@ -77,20 +80,20 @@ class Responsiveness {
    * one interaction had this duration by returning the first found.
    * @param {ResponsivenessEvent} responsivenessEvent
    * @param {LH.Trace} trace
-   * @return {EventTimingEvent}
+   * @return {EventTimingEvent|FallbackTimingEvent}
    */
   static findInteractionEvent(responsivenessEvent, {traceEvents}) {
     const candidates = traceEvents.filter(/** @return {evt is EventTimingEvent} */ evt => {
       // Examine only beginning/instant EventTiming events.
       return evt.name === 'EventTiming' && evt.ph !== 'e';
     });
 
-    if (candidates.length && !candidates[0].args.data.frame) {
+    if (candidates.length && !candidates.some(candidate => candidate.args.data?.frame)) {
```

Comment:
this should handle the seemingly impossible situation @paulirish was in, a newer Chrome but throwing at this point. AFAIK all `EventTiming` `'b'` and `'n'` events should have this format, but it's possible some somehow don't (empty string `frame`, maybe?).

Instead we just assert at least one candidate is possible of passing the frame equality check below.